js_modules/dg-docs/site/app/ComponentHeader.client.tsx

```diff
@@ -0,0 +1,9 @@
+'use client';
+
+import {ComponentHeader} from '@dagster-io/dg-docs-components';
+
+/**
+ * A wrapper re-export that applies `use client`, since the built output of the
+ * component library will remove any directives.
+ */
+export default ComponentHeader;
```

Comment:
The comment gives some info about this, but basically the Next.js `site` complains that `use client` is missing from the imported files because directives are stripped from built JS. This is the workaround suggested in Next.js docs to allow the component to be included in server-rendered pages.